<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaEstanciaUI</class>
 <widget class="QWidget" name="NuevaEstanciaUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>443</width>
    <height>556</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="Libro">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="Abierto">
      <property name="styleSheet">
       <string notr="true">#Abierto{border-image: url(:/Resources/Documentos/NuevaEstancia/documabierto.png);}</string>
      </property>
      <widget class="QPushButton" name="BotonRetroceder">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>340</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonAvanzar">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>350</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QStackedWidget" name="Documentos">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>271</width>
         <height>311</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="Estancia">
        <property name="styleSheet">
         <string notr="true">#Estancia{border-image: url(:/Resources/Documentos/NuevaEstancia/EstanciaNuevoDOC.png);}</string>
        </property>
        <widget class="QLabel" name="TituloMotivo">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>231</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Motivo de visita:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="TituloDuracion">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>231</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Duracion de estancia:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="DescTipoEstancia">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>231</width>
           <height>91</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="DescDuracEstancia">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>231</width>
           <height>91</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="PaisesPaso">
        <property name="styleSheet">
         <string notr="true">#PaisesPaso{border-image: url(:/Resources/Documentos/NuevaEstancia/paiseses de paso.png);}
</string>
        </property>
        <widget class="QLabel" name="DescPaisPaso">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>100</y>
           <width>211</width>
           <height>181</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Bienes">
        <property name="styleSheet">
         <string notr="true">#Bienes{border-image: url(:/Resources/Documentos/NuevaEstancia/bienes transportados.png);}</string>
        </property>
        <widget class="QLabel" name="DescBienes">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>231</width>
           <height>211</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Ocupacion">
        <property name="styleSheet">
         <string notr="true">#Ocupacion{
	
	border-image: url(:/Resources/Documentos/NuevaEstancia/ocupacion.png);
}</string>
        </property>
        <widget class="QLabel" name="DescOcupacion">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>231</width>
           <height>211</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Serif</family>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="Cerrado">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">#Cerrado{image: url(:/Resources/Documentos/NuevaEstancia/datosprevioscerrado.png);}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>55</number>
       </property>
       <property name="topMargin">
        <number>45</number>
       </property>
       <property name="rightMargin">
        <number>65</number>
       </property>
       <property name="bottomMargin">
        <number>45</number>
       </property>
       <item>
        <widget class="QPushButton" name="BotonAbrirLibro">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
